Asuka_src_1.1 のみに存在: Asuka
diff --color -u Asuka_src_1.1_old/Makefile Asuka_src_1.1/Makefile
--- Asuka_src_1.1_old/Makefile	2010-01-30 03:32:20.000000000 +0900
+++ Asuka_src_1.1/Makefile	2020-03-12 10:24:52.656154565 +0900
@@ -5,13 +5,13 @@
 O = o
 A = a
 C = c
-CPP = cpp
-CC = g++
-LD = g++
-GC = gcc
-CFLAGS = -O2 -I.
-CPPFLAGS =
-PGNAME = Asuka.exe
+CPP = clang++
+CC = clang++
+LD = clang++
+GC = clang
+CFLAGS = -I. -g
+CPPFLAGS = -DLINUX
+PGNAME = aska
 
 #
 # SRC Directory
Asuka_src_1.1 のみに存在: aska
diff --color -u Asuka_src_1.1_old/filepath.h Asuka_src_1.1/filepath.h
--- Asuka_src_1.1_old/filepath.h	2003-09-13 02:27:00.000000000 +0900
+++ Asuka_src_1.1/filepath.h	2020-03-11 23:23:46.460537970 +0900
@@ -6,8 +6,8 @@
 
 #pragma warning(disable:4786)
 
-#include <stdio.h>
-#include <stdlib.h>
+#include <cstdio>
+#include <cstdlib>
 
 #include <string>
 #include <list>
Asuka_src_1.1 のみに存在: filepath.o
diff --color -u Asuka_src_1.1_old/generator.h Asuka_src_1.1/generator.h
--- Asuka_src_1.1_old/generator.h	2003-09-13 02:28:00.000000000 +0900
+++ Asuka_src_1.1/generator.h	2020-03-11 23:24:28.960717476 +0900
@@ -6,8 +6,9 @@
 
 #pragma warning(disable:4786)
 
-#include <stdio.h>
-#include <stdlib.h>
+#include <cstdio>
+#include <cstdlib>
+#include <cstring>
 #include <iostream>
 
 #include <string>
Asuka_src_1.1 のみに存在: generator.o
Asuka_src_1.1 のみに存在: main.o
diff --color -u Asuka_src_1.1_old/module.h Asuka_src_1.1/module.h
--- Asuka_src_1.1_old/module.h	2010-01-10 21:41:08.000000000 +0900
+++ Asuka_src_1.1/module.h	2020-03-11 23:24:12.870649520 +0900
@@ -6,8 +6,8 @@
 
 #pragma warning(disable:4786)
 
-#include <stdio.h>
-#include <stdlib.h>
+#include <cstdio>
+#include <cstdlib>
 #ifndef LINUX
 #include <io.h>
 #endif
Asuka_src_1.1 のみに存在: module.o
diff --color -u Asuka_src_1.1_old/parser.h Asuka_src_1.1/parser.h
--- Asuka_src_1.1_old/parser.h	2003-09-13 02:30:00.000000000 +0900
+++ Asuka_src_1.1/parser.h	2020-03-11 23:23:49.550551019 +0900
@@ -6,8 +6,8 @@
 
 #pragma warning(disable:4786)
 
-#include <stdio.h>
-#include <stdlib.h>
+#include <cstdio>
+#include <cstdlib>
 #include <iostream>
 
 #include <string>
Asuka_src_1.1 のみに存在: parser.o
diff --color -u Asuka_src_1.1_old/scanner.h Asuka_src_1.1/scanner.h
--- Asuka_src_1.1_old/scanner.h	2003-09-13 02:31:00.000000000 +0900
+++ Asuka_src_1.1/scanner.h	2020-03-11 23:26:43.171284308 +0900
@@ -6,8 +6,9 @@
 
 #pragma warning(disable:4786)
 
-#include <stdio.h>
-#include <stdlib.h>
+#include <cstring>
+#include <cstdio>
+#include <cstdlib>
 #include <iostream>
 
 #include <string>
Asuka_src_1.1 のみに存在: scanner.o
diff --color -u Asuka_src_1.1_old/stringx.h Asuka_src_1.1/stringx.h
--- Asuka_src_1.1_old/stringx.h	2003-09-13 02:31:00.000000000 +0900
+++ Asuka_src_1.1/stringx.h	2020-03-11 23:23:51.060557399 +0900
@@ -6,9 +6,9 @@
 
 #pragma warning(disable:4786)
 
-#include <stdio.h>
-#include <stdlib.h>
-#include <ctype.h>
+#include <cstdio>
+#include <cstdlib>
+#include <cctype>
 #include <string>
 #include <list>
 #include <map>
Asuka_src_1.1 のみに存在: stringx.o
diff --color -u Asuka_src_1.1_old/table.h Asuka_src_1.1/table.h
--- Asuka_src_1.1_old/table.h	2003-09-13 02:31:00.000000000 +0900
+++ Asuka_src_1.1/table.h	2020-03-11 23:23:51.740560270 +0900
@@ -6,8 +6,8 @@
 
 #pragma warning(disable:4786)
 
-#include <stdio.h>
-#include <stdlib.h>
+#include <cstdio>
+#include <cstdlib>
 #include <iostream>
 #include <string>
 #include <list>
Asuka_src_1.1 のみに存在: table.o
diff --color -u Asuka_src_1.1_old/textmodule.h Asuka_src_1.1/textmodule.h
--- Asuka_src_1.1_old/textmodule.h	2003-09-13 02:32:00.000000000 +0900
+++ Asuka_src_1.1/textmodule.h	2020-03-11 23:23:52.330562760 +0900
@@ -7,8 +7,8 @@
 
 #pragma warning(disable:4786)
 
-#include <stdio.h>
-#include <stdlib.h>
+#include <cstdio>
+#include <cstdlib>
 
 #include <string>
 #include <list>
Asuka_src_1.1 のみに存在: textmodule.o
