--- z_tools/guigui00/stdlib_old.h	2010-01-10 12:12:18.000000000 +0900
+++ z_tools/guigui00/stdlib.h	2020-03-11 23:56:32.158840054 +0900
@@ -13,9 +13,6 @@
 #define	RAND_MAX	0x7fff
 #define srand(seed)			(void) (rand_seed = (seed))
 
-#define EXIT_SUCCESS		0
-#define EXIT_FAILURE		1
-
 int abs(int n);
 double atof(const char *s);
 int atoi(const char *s);
@@ -29,9 +26,6 @@
 
 void *malloc(unsigned int nbytes);
 void free(void *ap);
-void exit(int status);
-#define abort()		exit(EXIT_FAILURE)
-int system(const char *s);
 
 #if (defined(__cplusplus))
 	}
